ftfy/fixes.py

```diff
@@ -325,12 +325,12 @@ def fixup(match):
             except ValueError:
                 pass
         else:
-            # named entity
+            # This is a named entity; if it's a known HTML5 entity, replace
+            # it with the appropriate character.
             try:
-                text = entities[text[1:]]
+                return entities.html5[text[1:]]
             except KeyError:
-                pass
-        return text  # leave as is
```

Comment:
(Also, er, if you're tidying up here anyway, the fact that the function is defined inside the other function confused me for a little -- it has a variable `text` that has the same name as one in the enclosing scope, but AFAICT does not in reality use anything from that scope.)